assets/css/styles.css

```diff
@@ -1326,6 +1334,20 @@ a.no-style:visited {
     }
 }
 
+#video-modal .modal-xl {
+    width: 66.667%;
+}
+
+@media (max-width: 768px) {
+    #video-modal .modal-xl {
+        width: 100%;

```

Comment:
Just a detail. For some reason dialog container overflows on mobile devices.
Other modals doesn't have that problem. The easiest fix would be:

```css
width: calc(100vw - 15px);
```

but the reason of that overflow is somewhere else.